src/GitHub.InlineReviews/ViewModels/CommentViewModel.cs

```diff
@@ -130,22 +131,42 @@ protected void AddErrorHandler<T>(ReactiveCommand<T> command)
 
         async Task DoDelete(object unused)
         {
-            try
-            {
-                ErrorMessage = null;
-                IsSubmitting = true;
+            string deleteCommentMessageBoxText = VisualStudio.UI.Resources.DeleteCommentConfirmation;
+            string deleteCommentCaption = VisualStudio.UI.Resources.DeleteCommentConfirmationCaption;
+            
+            MessageBoxButton deleteCommentButton = MessageBoxButton.YesNo;
+            MessageBoxImage deleteCommentImage = MessageBoxImage.Question;
```

Comment:
@donokuda interested to know why you create variables for these things? Wouldn't it be easier just to do:

```
var result = MessageBox.Show(
    VisualStudio.UI.Resources.DeleteCommentConfirmation,
    VisualStudio.UI.Resources.DeleteCommentConfirmationCaption,
    MessageBoxButton.YesNo,
    MessageBoxImage.Question);
```

Reply:
Took care of this one.